fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -832,7 +833,7 @@ public class Config extends ConfigBase {
      * Parallel load fragment instance num in single host
      */
     @ConfField(mutable = true)
-    public static int load_parallel_instance_num = 1;
+    public static int load_parallel_instance_num = BackendCoreStat.getAvgNumOfHardwareCoresOfBe();
```

Comment:
I recommend changing this parameter to job granularity instead of changing the default value, which may cause additional resources to be used without the user's knowledge, resulting in stability problems.